qpext/go.mod

```diff
@@ -49,11 +49,11 @@ require (
 	go.uber.org/atomic v1.11.0 // indirect
 	go.uber.org/automaxprocs v1.5.3 // indirect
 	go.uber.org/multierr v1.11.0 // indirect
-	golang.org/x/net v0.20.0 // indirect
+	golang.org/x/net v0.21.0 // indirect
```

Comment:
We can bump to 0.23

Reply:
it is being done above with replace.

modifying them here might lead to unexpected changes when running go mod tidy.